packages/core/src/components/dropdown/dropdown-option/dropdown-option.scss

```diff
@@ -24,7 +24,9 @@
     }
 
     & button:focus {
-      @include tds-focus-state;
+      outline: 2px solid var(--tds-dropdown-option-focus);
+      box-shadow: inset 0 0 0 3px var(--tds-white);
+      outline-offset: -2px;
```

Comment:
Just wondering can we alter mixin tds-focus-state so it has this rules in as other component can use it or dropdown   option needed a tweak of global focus state?